python/ray/tests/test_basic.py

```diff
@@ -2649,8 +2649,6 @@ def f():
     output_lines = captured["out"]
     for i in range(200):
         assert str(i) in output_lines
```

Comment:
Unrelated to the test failure, but in this test we should really be checking that we don't get any unintended log messages (or duplicates). Especially since @stephanie-wang saw some duplicates recently.